app/views/content/life-as-a-teacher/chemistry/_article.html.erb

```diff
@@ -0,0 +1,127 @@
+    name: "Holly, science teacher",
+    large: true
+  ) %>
+  </section>
+
+  <section class="col col-720">
+    <h2 class="heading--box-blue">What you'll teach</h2>
+    <p>You'll teach the <a href="https://www.gov.uk/government/publications/national-curriculum-in-england-science-programmes-of-study/national-curriculum-in-england-science-programmes-of-study">national curriculum for chemistry</a>.</p>
+    <p>You could have the opportunity to teach or support physics and biology lessons throughout key stages 3 and 4.</p>
+    <p>When you teach 11 to 14 year olds (key stage 3), you'll cover themes from the 3 science disciplines of chemistry, biology and physics.</p>
+    <p>The chemistry themes are:</p>
+    <ul>
+      <li>the particulate nature of matter</li>
+      <li>atoms, elements and compounds</li>
+      <li>pure and impure substances</li>
+      <li>chemical reactions</li>
+      <li>energetics</li>
+      <li>the periodic table</li>
+      <li>materials</li>
+      <li>earth and atmosphere</li>
+    </ul>
+    <p>Themes you'll cover when you teach 14 to 16 year olds (key stage 4) will be more specific to chemistry, and include:</p>
+    <ul>
+      <li>atomic structure and the periodic table</li>
+      <li>structure, bonding and the properties of matter</li>
+      <li>chemical changes</li>
+      <li>energy changes in chemistry</li>
+      <li>rate and extent of chemical change</li>
+      <li>chemical analysis</li>
+      <li>chemical and allied industries</li>
+      <li>earth and atmospheric science</li>
+    </ul>
+    <p><a href="https://www.thenational.academy/teachers/programmes/chemistry-secondary-ks4-foundation-aqa/units">Explore what teaching a key stage 4 chemistry lesson would be like</a>.</p>
+    <p>As a secondary school chemistry teacher, you'll teach pupils aged 11 to 18. This means you'll always be expanding your knowledge and approach to meet pupils' needs. Once you have qualified teacher status (QTS) you can also teach any other subject, as long as you can show you have enough expertise.</p>
+  </section>
+
+  <section class="col col-space-m col-720">
+    <h2 class="heading--box-blue">Start your journey into teaching chemistry</h2>
+    <h3>Check your qualifications</h3>
+    <p>To train to teach chemistry in England, you'll need:</p>
+    <ul>
+      <li>GCSEs at grade 4 (C) or above in English and maths (and science if you want to teach primary)</li>
+      <li>a bachelor's degree in any subject</li>
+    </ul>
+    <p>Having relevant A levels can show your subject knowledge, if you do not have a degree in chemistry.</p>
+    <p>Chemistry teacher training courses have had successful applications from candidates with a range of degrees, including biomedical sciences, forensic science, pharmacy, and natural sciences.
+    <p>Find out about <a href="/train-to-be-a-teacher/qualifications-you-need-to-teach">the qualifications needed to teach</a>.</p>
+  </section>
```

Comment:
Missing non-UK expander on check your qualifications?

Reply:
@mariosh-e could we take a look at this pls, not sure how to put it in